lib/srv/db/cloud/meta.go

```diff
@@ -203,3 +221,18 @@ func describeRedshiftCluster(ctx context.Context, redshiftClient redshiftiface.R
 	}
 	return out.Clusters[0], nil
 }
+
+// describeElastiCacheCluster returns AWS ElastiCache Redis cluster for the
+// specified ID.
+func describeElastiCacheCluster(ctx context.Context, elastiCacheClient elasticacheiface.ElastiCacheAPI, replicationGroupID string) (*elasticache.ReplicationGroup, error) {
+	out, err := elastiCacheClient.DescribeReplicationGroupsWithContext(ctx, &elasticache.DescribeReplicationGroupsInput{
+		ReplicationGroupId: aws.String(replicationGroupID),
+	})
+	if err != nil {
+		return nil, common.ConvertError(err)
+	}
+	if len(out.ReplicationGroups) != 1 {
+		return nil, trace.BadParameter("expected 1 ElastiCache cluster for %v, got %s", replicationGroupID, out.ReplicationGroups)
```

Comment:
Maybe I'm missing something, but shouldn't we print the lenght `len(out.ReplicationGroups)` or use `%v` in the format to see all values?

Reply:
Good question. I copied from function above so didn't think about it too much. I feel we are passing the slice for debugging purpose. `%s` vs `%v`  probably not that much different. `%+v` or `%#v` may be better for a slice of structs.
Though, in this particular case, since `Id` is provided to the API, AWS should return either 0 or 1 item. 
I think I will go with `%#v`.